.gitignore

```diff
@@ -1,8 +1,13 @@
 .idea
-.vscode
 **/node_modules
 **/package-lock.json
 build
 .nyc_output
 coverage
-.DS_Store
\ No newline at end of file
+.DS_Store
+src/*.js
```

Comment:
looks like you build JS files using `tsc` command. Please see this extension https://marketplace.visualstudio.com/items?itemName=hbenl.vscode-mocha-test-adapter. It allows runnig  mocha tests without `sourceMap`, compiling and `launch.json` file